### ISSUE TITLE
Missing accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#16]
Hello! I noticed that the study group map is missing some accessibility features. It would be great if we could add some accessibility features to make the map more inclusive. Thank you!
